build.gradle

```diff
@@ -1,32 +1,40 @@
-plugins {
-    id 'java'
-    id 'maven-publish'
-    id 'pl.allegro.tech.build.axion-release' version '1.3.2'
-}
+group 'com.scmspain.karyon'
+
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'signing'
+apply plugin: 'com.bmuschko.nexus'
+apply plugin: 'io.codearte.nexus-staging'
+apply plugin: 'pl.allegro.tech.build.axion-release'
 
-scmVersion {
-    ignoreUncommittedChanges = false
+buildscript {
+    repositories {
+        mavenCentral()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.bmuschko:gradle-nexus-plugin:2.3.1'
+        classpath 'pl.allegro.tech.build:axion-release-plugin:1.3.2'
+        classpath "io.codearte.gradle.nexus:gradle-nexus-staging-plugin:0.5.3"
+    }
 }
 
-sourceCompatibility = 1.8
+apply from: 'gradle/publish.gradle'
 
-project.version = scmVersion.version
+sourceCompatibility = 1.8
 
 repositories {
-    mavenLocal()
-    maven {
-        credentials {
-            username project.hasProperty('artifactory_scm_user')? artifactory_scm_user: ''
-            password project.hasProperty('artifactory_scm_password')? artifactory_scm_password: ''
-        }
-
-        url "http://artifactory.scmspain.com/artifactory/repo"
-    }
     mavenCentral()
 }
 
-
 dependencies {
+    compile 'javax.ws.rs:jsr311-api:1.1.1'
+    compile 'com.netflix.karyon:karyon2-core:2.7.1'
+    compile 'com.netflix.karyon:karyon2-archaius:2.7.1'
+    compile 'com.netflix.karyon:karyon2-governator:2.7.1'
```

Comment:
I don't think we need karyon2-archaius and governator here. Also we should try to most karyon dependencies as runtime dependencies